engaging_s3invsync_temp.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+#SBATCH -p sched_any_quicktest
+#SBATCH --job-name=s3invsync
+#SBATCH --output=s3invsync-%j.out
+#SBATCH --error=s3invsync-%j.err
+#SBATCH --gres=gpu:1
+#SBATCH -t 02:00:00
+
+# Load Rust environment
+source $HOME/.cargo/env
+
+# Verify Rust installation
+rustc --version
+cargo --version
+
+export AWS_ACCESS_KEY_ID=<redacted>
+export AWS_SECRET_ACCESS_KEY=<redacted>
```

Comment:
Hi @aaronkanzer, instead of declaring this information in the sbatch script, you can define the environment variables in the login node, and then invoke the script using the following:
`sbatch --export=AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID,AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY engaging_s3invsync_temp.sh`

```suggestion
```